lib/opentok.js

```diff
@@ -839,6 +839,9 @@ OpenTok = function (apiKey, apiSecret, env) {
   *     <code>secure</code> (Boolean) &mdash; Whether the SIP media streams should be transmitted
   *     encrypted or not.
   *   </li>
+  *   <li>
+  *     <code>from</code> (String) &mdash; The number or string that will be sent to the final SIP number as the caller.
+  *   </li>
```

Comment:
This is missing a lot of the details from the REST API documentation (see https://tokbox.com/developer/rest/#sip_call). I think some of the descriptions for other options are also missing details.

Also, can we please note that all of the properties of the `options` parameter are optional:

```@param options {Object} An optional options object with the following properties (all of which are optional):```